### PR TITLE
Cardy update

### PR DESCRIPTION
**Pull request by davidpmccormick**
Sorry for the large PR; I didn't want to give you one PR per component that went in, but there is a commit per component.

__Adding__
- Avatar
- BetaBar
- BetaMessage
- ErrorPage
- ImagePlaceholder
- NewsletterPromo
- NewsletterSignup
- StatusIndicator

__Fixing__
- VenueHours
- Footer

### CODE REVIEW

File: cardigan/stories/components/BetaMessage.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import Readme from '../../../common/views/components/BetaMessage/README.md';
+import { ThemeProvider } from 'styled-components';
+import theme from '../../../common/views/themes/default';
+
+const stories = storiesOf('Components', module);
+
+stories.add(
+  'BetaMessage',
+  () => (
+    <ThemeProvider theme={theme}>
```
Comment (jamesgorrie) [inline cardigan/stories/components/BetaMessage.js]: Maybe we should wrap the whole story book in the theme?

File: cardigan/stories/components/ErrorPage.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <GlobalAlertContext.Provider value={''}>
+          <ErrorPage statusCode={404} />
+        </GlobalAlertContext.Provider>
+      </OpeningTimesContext.Provider>
+    </ThemeProvider>
+  ),
+  {
+    info: Readme,
+  }
+);
```
Comment (jamesgorrie) [inline cardigan/stories/components/ErrorPage.js]: Seems an odd component to have in cardigan as it contains everything, which we don't really do anywhere?
Would it make more sense to have this separated into its separate components?

File: cardigan/stories/components/ImageGallery.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      id="cardy"
+      isStandalone={boolean('Is standalone?', false)}
+    />
+  );
+};
+
+const stories = storiesOf('Components', module);
+stories.add('ImageGallery', ImageGalleryExample, {
+  info: Readme,
+  isFullScreen: true,
```
Comment (jamesgorrie) [inline cardigan/stories/components/ImageGallery.js]: How and when do we decide about this? Isn't it an option in the interface?

File: cardigan/stories/components/BetaMessage.js (hunk 1 above)
Comment (davidpmccormick) [reply cardigan/stories/components/BetaMessage.js]: That sounds like a good shout.

File: cardigan/stories/components/ErrorPage.js (hunk 2 above)
Comment (davidpmccormick) [reply cardigan/stories/components/ErrorPage.js]: P(ossi|roba)bly.

File: cardigan/stories/components/ImageGallery.js (hunk 3 above)
Comment (davidpmccormick) [reply cardigan/stories/components/ImageGallery.js]: Our default decorator adds 30px of padding around each component in Storybook, to make it look nice. But when a component is full-bleed, removing this padding gives a more honest representation of the component.

**Pull request merged**